### ISSUE TITLE
Failed to retrieve Access Token: ('Error trying to extract API version: %s', TypeError('get() takes 1 positional argument but 2 were given'))

### ISSUE THREAD

**Issue by TheAlpiner** [rospogrigio/daikin_residential#84]
Tried to connect my new Daikin heathpump. The installation was straightforward but when i put in my username and password i get an "Failed to retrieve access token" message. Below is a copy from the core log:

```
Logger: custom_components.daikin_residential.config_flow
Source: custom_components/daikin_residential/config_flow.py:72
Integration: Daikin Residential Controller ([documentation](https://github.com/rospogrigio/daikin_residential/), [issues](https://github.com/rospogrigio/daikin_residential/issues))
First occurred: 00:09:03 (4 occurrences)
Last logged: 09:54:23

Failed to retrieve Access Token: ('Error trying to extract API version: %s', TypeError('get() takes 1 positional argument but 2 were given'))
```
Following the docs it does not seem that i can do much wrong, Anyone experienceing the same?

**Comment by rospogrigio** [comment]
This integration supports only A/C devices, not heat pumps.
You might want to have a look at this integration: https://github.com/speleolontra/daikin_residential_altherma
Bye!

**Comment by TheAlpiner** [comment]
What is the difference. I live in a cold climate we call it heathpump, i
have an apartement in spain there it is AC. Bottomline it is the same. My
Daikin unit is the same as the others that have got this integration to
work :-)

tir. 16. aug. 2022 kl. 11:21 skrev rospogrigio ***@***.***>:

**Comment by Neo2SHYAlien** [comment]
Obviously the API is different :)

**Comment by TheAlpiner** [comment]
HM! OK, i hear your statements but my controller is the BRP069C4x that many has got to work reading trough several threads. this for example: https://community.home-assistant.io/t/daikin-ac-integration-brp069c4x/270936/284

So why my API obviously is different is hard to see for me. But again I am just a humble guy :-). 

Yes i should probably have put in the type in the first message sorry about that. Anyway thank you for the help and insights ;-)

**Comment by rospogrigio** [comment]
Well, that changes the scenario, the BRP069C4x is supported.

**Comment by TheAlpiner** [comment]
Hi, that was my understanding that it was supported. I also use the ONECTA app. I have seen other threads also mentioning this app. But it is the same cloud service and their unit works so i thought mine should to. 

No i have not tried the debug script. I will do :-)

**Comment by TheAlpiner** [comment]
Hi, just an update on this issue. I have been fighting a bit to fix this without any luck. I configured a blank and fresh HA instance and installed the integration. And everything works as a charm. So my problem is not the integration but something i have installed or configured on the HA instance is causing this. I found this very strange. Could another integration interfere with this one? I can not run the diagnostics because i never get passed the login box (at least i do not now how to run it from a integration that is not activated. I really do not want to scrap my existing one and start fresh so tips on how to troubleshoot this will be highly valued 🙂

**Comment by rospogrigio** [comment]
Did you try to launch the tokensaver.py script you find in the tools folder? That might help understand what's going on

**Comment by TheAlpiner** [comment]
I will try 🙂👍

**Comment by TheAlpiner** [comment]
After som more research i decided to go the hard way. I disabled one integration at a time until i found the one tha failed. And i found the failing one. And it is this one:

https://github.com/h4de5/home-assistant-toshiba_ac

I disabled it, reinstalled your integration and the Daiking unit popped up correctly. After that i activated the Toshiba integration and now everything seems to work… But something is wrong. I guess in the Toshiba one…
